x-pack/plugins/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -400,46 +384,17 @@ export const deletePackagePolicyHandler: RequestHandler<
   const soClient = coreContext.savedObjects.client;
   const esClient = coreContext.elasticsearch.client.asInternalUser;
   const user = appContextService.getSecurity()?.authc.getCurrentUser(request) || undefined;
-  const logger = appContextService.getLogger();
 
   try {
-    try {
-      const packagePolicies = await packagePolicyService.getByIDs(
-        soClient,
-        request.body.packagePolicyIds,
-        { ignoreMissing: true }
-      );
-
-      if (packagePolicies) {
-        await packagePolicyService.runExternalCallbacks(
-          'packagePolicyDelete',
-          packagePolicies,
-          context,
-          request
-        );
-      }
-    } catch (error) {
-      logger.error(`An error occurred executing external callback: ${error}`);
-      logger.error(error);
-    }
-
     const body: PostDeletePackagePoliciesResponse = await packagePolicyService.delete(
+      context,
+      request,
```

Comment:
I had to add `context` and `request` because the callbacks require them. However, some places the package policy handlers are called outside of a request cycle (eg. during startup) where `request` is not available. 

Any suggestions?